lib/phoenix_token_auth/registration_flow.ex

```diff
@@ -0,0 +1,28 @@
+defmodule PhoenixTokenAuth.RegistrationFlow do
+  alias PhoenixTokenAuth.Mailer
+  alias PhoenixTokenAuth.Util
+
+  def user_created_by_username?(changeset) do
+    if changeset.valid? do
+      Util.repo.transaction(fn ->
+        case Util.repo.insert(changeset) do
```

Comment:
yep, please do. i think it crept in there during earlier refactoring. but it's definitely good to reduce complexity and remove it.